blob/drivertest/drivertest.go

```diff
@@ -1368,6 +1371,86 @@ func testMetadata(t *testing.T, newHarness HarnessMaker) {
 	}
 }
 
+// testMD5 tests reading MD5 hashes via List and Attributes.
+func testMD5(t *testing.T, newHarness HarnessMaker) {
+	ctx := context.Background()
+
+	// Define two blobs with different content; we'll write them and then verify
+	// their returned MD5 hashes.
+	const aKey, bKey = "blob-for-md5-aaa", "blob-for-md5-bbb"
+	aContent, bContent := []byte("hello"), []byte("goodbye")
+	hash := md5.New()
```

Comment:
I think you can use the top-level function `md5.Sum` instead of these three lines.

Reply:
Done, although it returns a `[16]byte` so I needed to add `[:]` down below in `bytes.Equal`.